Sources/PublishCLICore/WebsiteRunner.swift

```diff
@@ -1,20 +1,117 @@
 /**
-*  Publish
-*  Copyright (c) John Sundell 2019
-*  MIT license, see LICENSE file for details
-*/
+ *  Publish
+ *  Copyright (c) John Sundell 2019
+ *  MIT license, see LICENSE file for details
+ */
 
 import Foundation
 import Files
 import ShellOut
+import FileWatcher
 
 internal struct WebsiteRunner {
+    static let normalTerminationStatus = 15
+    static let debounceDuration = 3 * NSEC_PER_SEC
+    static let runLoopInterval: TimeInterval = 0.1
+    static let exitMessage = "Press CTRL+C to stop the server and exit"
```

Comment:
I was unable to get the folder watching to work while maintaining the `Press ENTER to stop the server and exit` behaviour - I replaced `readLine()` with a non-blocking implementation (checking standard input on each iteration of the `while` loop), but that interfered with the watching functionality for an unknown reason.